config/environments/production.rb

```diff
@@ -22,36 +22,29 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # Compress CSS using a preprocessor.
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-  # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
-
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
   # config.action_controller.asset_host = 'http://assets.example.com'
 
   # Specifies the header that your server uses for sending files.
   # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
-  # Store uploaded files on the local file system (see config/storage.yml for options)
+  # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
-  # Mount Action Cable outside main process or domain
+  # Mount Action Cable outside main process or domain.
   # config.action_cable.mount_path = nil
   # config.action_cable.url = 'wss://example.com/cable'
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
-
-  # Handle STS here instead of Apache, or Rails duplicates header contents
-  # Also unset cache-control header in HTTPS vhost for same reason
-  config.ssl_options = { hsts: { preload: true } }
```

Comment:
This was important production configuration that we want to retain. This should be the same across all our Rails applications. I usually put a leading comment line like `# Local changes` or similar when having to edit config values in the middle of config files (rather than just things appended at the end) to differentiate what is our intentional deviation from defaults.

Would that help with things like this and the accidental restoration the Turbolinks gem in one of the other recent Rails upgrade PRs? I'm not sure if you're looking at diffs of any of the files Rails changes or not as the vast majority haven't been customized and Rails is free to update as it sees fit. I think it's just these environment config files and Gemfile that require extra attention :thinking: 

I might want to make an issue or column in the application migration spreadsheet to check that these are all intact after Rails upgrades.

Reply:
I don't think this is a customization or anything, nor do I think I removed it in the `rails app:update` process. I think I just removed to deal with local errors. I will restore it.

Reply:
I see now that it is a customization, and probably was removed because of `rails app:update` and I assumed it was an old Rails 5 setting. I think I made this change elsewhere, too.

Reply:
fixed